docs/howto/catalog_exports.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Data Catalogs Export
+description: This section explains how lakeFS can integrate with external Data Catalogs via metastore update operations. 
+parent: How-To
+---
+
+# Data Catalogs Export
+
+{% include toc_2-3.html %}
+
+## About Data Catalogs Export
+
+Data Catalog Export is all about integrating query engines (like Spark, AWS Athena, Presto, etc.) with lakeFS.
+
+Data Catalogs (such as Hive Metastore or AWS Glue) store metadata for services (such as Spark, Trino and Athena). They contain metadata such as the location of the table, information about columns, partitions and much more.
+
+With Data Catalog Exports, one can leverage the versioning capabilities of lakeFS in external data warehouses and query engines to access tables with branches and commits. 
+
+At the end of this guide, you will be able to query lakeFS data from Athena, Trino and other catalog-dependent tools:
+
+```sql
+use main;
+use my_branch; -- any branch
+use v101; -- or tag
+
+SELECT * FROM users 
+INNER JOIN events 
+ON users.id = events.user_id; -- SQL stays the same, branch or tag exist as schema
+```
+
+## How it works 
+
+There are several well known formats that exist today that could help export existing tables in lakeFS into a "native" object store representation
+which don't require copying the data outside of lakeFS.
```

Comment:
Usually the active voice is easier to read than the passive voice.  So here I would prefer something like
```suggestion
Several well known formats exist today let you export existing tables in lakeFS into a "native" object store representation
which does not require copying the data outside of lakeFS.
```